Resources/Prototypes/_Sunrise/Entities/Objects/Specific/Robotics/borg_tools.yml

```
@@ -239,6 +239,36 @@
     - ReagentId: DexalinPlus
       Quantity: 2
 
+- type: entity
+  name: gorlax robot hypospray
+  parent: HypoBorgStandard
+  description: A borg version of hypospray that automatically regenerates epinephrine.
+  id: HypoBorgStandardSyndicate
+  categories: [ HideSpawnMenu ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)